cmake-tool/helpers/check_arch_compiler.cmake

```diff
@@ -14,7 +14,7 @@ function(check_arch_clang)
     elseif("${KernelSel4Arch}" STREQUAL "aarch64")
         string(REGEX MATCH "^aarch64" correct_triple ${TRIPLE})
     elseif("${KernelSel4Arch}" STREQUAL "riscv32")
-        string(REGEX MATCH "^riscv32" correct_triple ${TRIPLE})
+        string(REGEX MATCH "^riscv64" correct_triple ${TRIPLE})
```

Comment:
This should encode that both riscv64 and riscv32 toolchains can handle riscv32 architectures but will now reject riscv32 toolchains.  Is it not possible to get a riscv32 toolchain anymore?

Reply:
good catch, it's not the default but it's definitely possible.